### ISSUE TITLE
How to render complex components in Playground

### ISSUE THREAD

**Issue by angel-git** [pedronauck/docz#162]
## Question
Hello,
I would like to pass some variables to my components and use it inside `<Playground>` element as in the sample from 
https://github.com/pedronauck/docz/blob/master/examples/basic/src/components/Alert.mdx#L37

So in my case I have:
```
<Playground>
{() => {
     const subprojects = []
     return <Project label="SomeLabel" projects="{subprojects}" />
  }}
</Playground>
```

But this just renders as:
<img width="590" alt="screen shot 2018-07-23 at 09 46 26" src="https://user-images.githubusercontent.com/1565058/43063811-4b86d3e8-8e5d-11e8-98ee-e64d43c1c893.png">

These are my devDepedencies:
```
"docz": "^0.5.8",
"docz-plugin-css": "^0.5.7",
"enzyme": "^3.3.0",
"enzyme-adapter-react-16": "^1.1.1",
"react-test-renderer": "^16.4.1",
"typescript": "^2.9.2"
```

**Comment by pedronauck** [comment]
Hi @angel-git, So let me know, what you want to be rendered? Because this is the expected behavior of `<Playground>` when you use some kind of function as a child

**Comment by angel-git** [comment]
@pedronauck I would like to use the `const subprojects = []`variable inside my `<Project />` element.
Let me rephrase then, how to pass variables to my component from mdx file?

**Comment by pedronauck** [comment]
You can import variables from another file, like this:

```
import { subprojects } from './whatever'

<Playground>
  <Project label="SomeLabel" projects="{subprojects}" />
</Playground>
```

Or just use like you did above using children as a function and define the variables inside it scope!

**Comment by angel-git** [comment]
sorry, it's me again :)

that's exactly what's is wrong, the variables are not resolved:

my project component:
```
function Project({ label, projects }: IProps) {

    return (
        <div>{label} {projects}</div>
    );
```

on mdx:
```
<Playground>
{() => {
     const someVar = 'someValue'
     return <Project label="SomeLabel" projects="{someVar}" />
  }}
</Playground>
```

<img width="613" alt="screen shot 2018-07-23 at 21 48 00" src="https://user-images.githubusercontent.com/1565058/43099322-1bf0b30a-8ec2-11e8-9668-9311a4c1498d.png">

Is printing someVar as string but the value of the variable is not resolved

**Comment by mAAdhaTTah** [comment]
@angel-git This is a bug in your JSX:

```js
return <Project label="SomeLabel" projects="{someVar}" />

// should be....
return <Project label="SomeLabel" projects={someVar} />
```

then `""` around `{someVar}` makes it get passed in as a string, rather than the value.

**Comment by angel-git** [comment]
ahhh how retarded I'm! thanks!
